src/pr-info.ts

```diff
@@ -319,32 +319,18 @@ type FileLocation = ({
 }) & { filePath: string };
 
 function categorizeFile(filePath: string): FileLocation {
-    const typeDefinitionFile = /^types\/([^\/]+)\/(.*)\.d\.ts$/i;
-    // https://regex101.com/r/QfAfRn/1
-    const typeTestFile = /^types\/([^\/]+)\/(.*)\.tsx?$/i;
-    const typeOtherFile = /^types\/([^\/]+)\/(.*)$/i;
-    let match;
-    if (match = typeDefinitionFile.exec(filePath)) {
-        return { filePath, kind: "definition", package: match[1] };
-    } else if (match = typeTestFile.exec(filePath)) {
-        return { filePath, kind: "test", package: match[1] };
-    } else if (match = typeOtherFile.exec(filePath)) {
-        return { filePath, kind: "package-meta", package: match[1] };
-    } else {
-        return { filePath, kind: "infrastructure" };
+    const match = /^types\/(.*?)\/.*?[^\/](?:\.(d\.ts|tsx?))?$/.exec(filePath);
```

Comment:
Can you include a link like: `https://regex101.com/r/QfAfRn/1` which shows the regex with some passing and failing tests please

Reply:
Done, though for some reason the highlights on this thing are broken when I switch to JS mode.  But the match info on the RHS is fine, so I left it in.